libvips/foreign/dzsave.c

```diff
@@ -1495,7 +1497,7 @@ tile_name( Layer *layer, int x, int y )
 			int size = VIPS_MIN( dz->tile_size, 
 				layer->width - x * dz->tile_size );
 
-			vips_snprintf( dirname2, VIPS_PATH_MAX, "%d,", size );
+			vips_snprintf( dirname2, VIPS_PATH_MAX, "%d", size );
```

Comment:
I believe the trailing comma is needed for iiif2 -- I think it means "ysize == xsize" (I think).

Reply:
@jcupitt Just had another look and it turns out you're right, it is needed for IIIFv2. I'll amend the PR.